types.json

```diff
@@ -12,7 +12,7 @@
   "AssetQuantity": "Quantity",
   "CashQuantity": "Quantity",
   "USDQuantity": "Quantity",
-  "LiquidityFactor": "Factor",
+  "LiquidityFactor": "(u128)",
```

Comment:
i don't think this is valid json due to reused key in the same map `LiquidityFactor`